components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -78,48 +80,44 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
         this.store.select(updateStatus),
         this.store.select(getProjectStatus)
       ]).pipe(
-        takeUntil(this.isDestroyed),
-        map(([gStatus, uStatus, gpStatus]) => {
-          const routeId = this.route.snapshot.paramMap.get('ruleid');
-          this.isLoading =
-            routeId
```

Comment:
cleanup, if routeId is null, this is false. Still the case below, without the ternary operator.